src/overlays/actors/ovl_En_Wdhand/z_en_wdhand.c

```diff
@@ -5,6 +5,7 @@
  */
 
 #include "z_en_wdhand.h"
+#include "objects/object_wdhand/object_wdhand.h"
```

Comment:
Nit: this `#include` is unnecessary, since it's also included via `#include "z_en_wdhand.h"`